go.mod

```diff
@@ -21,15 +21,16 @@ require (
 	contrib.go.opencensus.io/exporter/aws v0.0.0-20181029163544-2befc13012d0
 	contrib.go.opencensus.io/exporter/stackdriver v0.12.1
 	contrib.go.opencensus.io/integrations/ocsql v0.1.4
-	github.com/Azure/azure-amqp-common-go/v2 v2.1.0
+	github.com/Azure/azure-amqp-common-go/v3 v3.0.0
 	github.com/Azure/azure-pipeline-go v0.2.1
-	github.com/Azure/azure-sdk-for-go v30.1.0+incompatible
-	github.com/Azure/azure-service-bus-go v0.9.1
+	github.com/Azure/azure-sdk-for-go v37.1.0+incompatible
```

Comment:
Should I upgrade azure-sdk-for-go to the latest version? https://github.com/Azure/azure-sdk-for-go/releases/tag/v42.3.0

Reply:
Sure, might as well.

Reply:
Let's do it in a separate PR though.